numpy/core/src/npymath/npy_math_private.h

```diff
@@ -536,4 +540,11 @@ typedef union {
 } __npy_clongdouble_to_c99_cast;
 #endif /* !NPY_USE_C99_COMPLEX */
 
+/* Intel C for Windows uses POW for 64 bits longdouble*/
```

Comment:
Shouldn't this go into the existing blacklist at <https://github.com/numpy/numpy/blob/master/numpy/core/src/private/npy_config.h#L23>?